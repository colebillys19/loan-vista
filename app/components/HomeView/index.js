/**
 * HomeView
 * @description ...
 */

import React from 'react';
// import PropTypes from 'prop-types';

const HomeView = () => <div>HomeView</div>;

HomeView.propTypes = {};

export default HomeView;
