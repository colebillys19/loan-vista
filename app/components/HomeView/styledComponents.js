import styled from 'styled-components';

export const HomeViewWrapper = styled.div``;
