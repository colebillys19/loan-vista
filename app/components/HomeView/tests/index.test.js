import React from 'react';
import { render } from 'react-testing-library';

import HomeView from '../index';

describe('<HomeView />', () => {
  it.skip('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(<HomeView />);
    expect(spy).not.toHaveBeenCalled();
  });

  it.skip('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<HomeView />);
    expect(firstChild).toMatchSnapshot();
  });
});
