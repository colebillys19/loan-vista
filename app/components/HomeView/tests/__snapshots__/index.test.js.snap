// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView /> Should render and match the snapshot 1`] = `
<section
  class="styledComponents__StyledSection-gk85wv-0 eyjsLj"
>
  <div
    style="padding: 1rem;"
  >
    HomeView
  </div>
</section>
`;
